pkg/build/strategies/onbuild/onbuild.go

```diff
@@ -88,7 +89,7 @@ func (builder *OnBuild) SourceTar(config *api.Config) (io.ReadCloser, error) {
 // Build executes the ONBUILD kind of build
 func (builder *OnBuild) Build(config *api.Config) (*api.Result, error) {
 	if config.BlockOnBuild {
-		return nil, fmt.Errorf("builder image uses ONBUILD instructions but ONBUILD is not allowed")
+		return nil, errors.New("builder image uses ONBUILD instructions but ONBUILD is not allowed")
```

Comment:
that are we are being pedantic about not using fmt.Errorf when there is no formatting to be done.